packages/server/src/db/index.js

```diff
@@ -685,12 +681,9 @@ async function getAgenciesSubscribedToDigest(asOf) {
             SELECT
                 a.id,
                 a.name,
-                string_agg(DISTINCT aec.code,
-                    '|') AS codes,
-                string_agg(DISTINCT k.search_term,
-                    '|') AS term,
-                string_agg(DISTINCT u.email,
-                    ' | ') AS emails
+                array_agg(DISTINCT aec.code) AS codes,
+                array_agg(DISTINCT k.search_term) AS term,
+                array_agg(DISTINCT u.email) AS emails
```

Comment:
This isn't well-though-out enough to merit a requested change, but it might be possible (and better for performance) to drop the `DISTINCT` usage here, since it doesn't really matter (?) whether the same code/term/email potentially gets matched multiple times in the `JOIN`s further down. It would potentially yield a larger search subject in the regex operations, but that may well be better than enforcing uniqueness here.

As I said, this is just a quick thought, and I'm not convinced either way would have a major impact on performance, but just wanted to note it for your consideration.